### DISCUSSION TITLE
Auto back up feature

### DISCUSSION THREAD

**Discussion by thevishalsingh777** [fast4x/RiMusic#3358] [Ideas]
I think there should be a auto back up feature which backups the data every day or week or month. There should also be feature to auto delete previous backups

**Comment by fast4x** [comment]
What do you need an automatic backup for if it might fail due to lack of space?

**Reply by thevishalsingh777** [reply]
Lack of space for a 3 mb file? I think u misunderstood, I just want to automate the "save to backup" option instead of manually doing it every now and then.

**Reply by fast4x** [reply]
So you mean that next backup overwrite previous...

**Reply by thevishalsingh777** [reply]
Yep

**Reply by aneesh1122** [reply]
> So you mean that next backup overwrite previous...

Let there be a file called 'autosave.db' in RiMusic folder and it will be overwritten and updated periodically.
